lib/marked-renderer.js

```diff
@@ -1,7 +1,17 @@
 const { marked } = require('marked')
 
-/** A custom renderer that avoids wrapping `<img>` tags in `<p>` tags */
+/**
+ * Custom markdown renderer that avoids wrapping `<img>` image
+ * tags (or code with syntax highlighting) in `<p>` paragraphs
```

Comment:
This isn't quite right any more – as far as I know the `code` method has nothing to do with avoiding the code blocks being wrapped in paragraphs?

Suggest either adding comment blocks to the individual methods or changing to something like:

```suggestion
 * Custom markdown renderer that:
 * - avoids wrapping `<img>` image tags in `<p>` paragraphs
 * - adds tabindex="0" to code blocks so they can be focused and scrolled with
 *   the keyboard
```

Reply:
Ahh let me dig into why I adjusted the comment

I found that "new lines" in code blocks were triggering another `.code()` call as "code within code"

The first `.code()` call gets `<pre><code>` and have a syntax language set
Subsequent `.code()` could skip straight to `.html()` and don't have a syntax language

Hopefully it's something we fixed (or will fix) in:

* https://github.com/alphagov/govuk-design-system/pull/2565
* https://github.com/alphagov/govuk-design-system/pull/2578

We might both be right

Reply:
Yeah it was related new lines so the original comment has been moved to:

* https://github.com/alphagov/govuk-design-system/pull/2578#discussion_r1113608969

This PR is now stacked on top and adds the comment:

```mjs
// Ensure code blocks can be focused and scrolled
// with the keyboard via `tabindex="0"`
```